gym_xiangqi/envs/xiangqi_env.py

```diff
@@ -149,17 +214,28 @@ def init_pieces(self):
                 elif piece_id > 0:
                     self.agent_piece[piece_id] = init(self.agent_color, r, c)
 
-    def get_possible_actions(self):
+    def get_possible_actions(self, player):
```

Comment:
Hmm I think keeping 2 arrays, 1 for agent moves, 1 for enemy moves will be better than clearing the array each time. Eventually we wouldn't want to clear the entire array also, but remove the possible actions from that one piece that is moved.